src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostOptions.cs

```diff
@@ -53,6 +53,14 @@ internal static class RemoteHostOptions
             storageLocations: new LocalUserProfileStorageLocation(InternalFeatureOnOffOptions.LocalRegistryPath + nameof(OOP64Bit)));
 
         public static readonly Option<bool> RemoteHostTest = new Option<bool>(nameof(InternalFeatureOnOffOptions), nameof(RemoteHostTest), defaultValue: false);
+
+        public static readonly Option<bool> EnableConnectionPool = new Option<bool>(
+            nameof(InternalFeatureOnOffOptions), nameof(EnableConnectionPool), defaultValue: true,
+            storageLocations: new LocalUserProfileStorageLocation(InternalFeatureOnOffOptions.LocalRegistryPath + nameof(EnableConnectionPool)));
+
+        public static readonly Option<int> MaxPoolConnection = new Option<int>(
+            nameof(InternalFeatureOnOffOptions), nameof(MaxPoolConnection), defaultValue: 15,
```

Comment:
I'm guessing this number was chosen arbitrarily? I don't mind if it is, but I would appreciate a comment to that effect just so when somebody else ever thinks about changing this number they can be less afraid.

Reply:
sure